x-pack/plugin/core/src/main/java/org/elasticsearch/license/LicenseService.java

```diff
@@ -184,7 +185,7 @@ static CharSequence buildExpirationMessage(long expirationMillis, boolean expire
     }
 
     private void populateExpirationCallbacks() {
-        expirationCallbacks.add(new ExpirationCallback.Pre(days(7), days(25), days(1)) {
+        expirationCallbacks.add(new ExpirationCallback.Pre(days(0), days(25), days(1)) {
```

Comment:
I can open this as a separate PR, but noticed that a previous refactoring removed a no longer applicable callback here, but did not adjust this warning callback, so that we currently don't emit any warnings to the log from 7 to 0 days before license expiration.